src/Device/Driver/AR62xx.cpp

```diff
@@ -766,17 +602,18 @@ bool AR62xxDevice::PutStandbyFrequency(RadioFrequency frequency,
 /*
  * Assign the selected port on Object construction
  */
-static Device *AR62xxCreateOnPort(const DeviceConfig &config, Port &comPort)
-{
-  return new AR62xxDevice(comPort);
+static Device *AR62xxCreateOnPort(const DeviceConfig &config, Port &comPort) {
+  Device *dev = new AR62xxDevice(comPort);
+  return dev;
```

Comment:
Why split in two lines? That line was just fine. (Except for the wrong parameter name.)